.github/workflows/ci.yml

```diff
@@ -67,39 +67,59 @@ jobs:
 
           # Build optee_os and optee_client for qemu_v8
           ./build_optee_libraries.sh $HOME
-
-          # Setup environment
           export OPTEE_DIR=$HOME
-          source environment
 
-          # Build OP-TEE Rust examples for Arm 64-bit both host and TA
+          # Build OP-TEE Rust examples for Arm 32-bit both host and TA
+          export ARCH_HOST=arm
+          export ARCH_TA=arm
+          source environment
           make -j`nproc`
 
-          # Build OP-TEE Rust examples for Arm 64-bit host and 32-bit TA
-          export TA_DEV_KIT_DIR=~/optee_os/out/arm-plat-vexpress/export-ta_arm32/
-          export CROSS_COMPILE_HOST=$CROSS_COMPILE64
-          export CROSS_COMPILE_TA=$CROSS_COMPILE32
-          export TARGET_HOST="aarch64-unknown-linux-gnu"
-          export TARGET_TA="arm-unknown-linux-gnueabihf"
+          # Build OP-TEE Rust examples for Arm 64-bit both host and TA
+          unset ARCH_TA
+          unset ARCH_HOST
+          source environment
           make clean && make -j`nproc`
+      - name: Run tests for Arm 64-bit both host and TA
+        run: |
+          apt update && apt install libslirp-dev -y
+          source environment
+          (cd ci && ./ci.sh)
+  build-and-test-examples-for-std-TAs:
+    runs-on: ubuntu-latest
+    container: teaclave/teaclave-trustzone-sdk-build:0.3.0
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Setting up $HOME
+        run: |
+          cp /root/.bashrc $HOME/.bashrc &&
+          ln -sf /root/.rustup ~/.rustup &&
+          ln -sf /root/.cargo ~/.cargo
+      - name: Building Arm 64-bit both host and TA (with STD enabled)
```

Comment:
Can we support 32-bit TAs with STD enabled? I can see that we support 32-bit OP-TEE custom target for STD.

Reply:
Yes we support 32-bit TAs with std, I'll add a CI job for that.